parm/gdas/staging/atm_berror_bump.yaml.j2

```
@@ -0,0 +1,23 @@
+{% set src_dir = BERROR_DATA_DIR %}
+{% set dest_dir = DATA ~ '/berror' %}
+{% set ftype_list = ['cor_rh', 'cor_rv', 'stddev'] %}
+{% set nprocs = ntiles * layout_x * layout_y %}
+
+mkdir:
+- '{{ dest_dir }}'
+
+copy:
+{% for ftype in ftype_list %}
+   {% set prefix = BERROR_DATE | to_fv3time ~ '.' ~ ftype %}
+   {% set fname = prefix ~ '.coupler.res' %}
+- ['{{ src_dir }}/{{ fname }}', '{{ dest_dir }}/{{ fname }}']
+   {% for itile in range(1,7) %}
+      {% set fname = prefix ~ '.fv_tracer.res.tile' ~  itile ~ '.nc' %}
+- ['{{ src_dir }}/{{ fname }}', '{{ dest_dir }}/{{ fname }}']
```

Just a note here.
If the destination ends in `/`, it is assumed that the destination filename is the same as the source filename.  In that case, the above can be simplified as:
```suggestion
- ['{{ src_dir }}/{{ fname }}', '{{ dest_dir }}/']
```